material-color-utilities/build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+// TODO: Remove this after https://youtrack.jetbrains.com/issue/KTIJ-19369 is resolved.
+@Suppress("DSL_SCOPE_VIOLATION")
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin)
+}
+
+android {
+    compileSdk = 33
```

Comment:
I'm sure you don't intended on modifying this module much, but we do 
`compileSdkVersion = rootProject.compileSdkVersion` to prevent version mismatches

Reply:
See? This is why we code review